### ISSUE TITLE
Organisation name that is 8 characters or longer causes CloudFormation stack to fail

### ISSUE THREAD

**Issue by kangsoon93** [awslabs/aws-serverless-data-lake-framework#239]
**Describe the bug**
When setting organisation name to be 8 characters or longer, deployment of CloudFormation stack fails. Under v1.0, organisation name requirements was 9 characters or less. 

**To Reproduce**
1. Under 'Deploying the Foundations' step of the workshop, set the organisation name to 8 characters or longer.
3. Run git push to Codecommit as per instructions.
4. Error occurs in the CloudFormation substack sdlf-domain-datalake-dev-rForecourt-XXXXXXXX.

**Expected behavior**
Properties validation failed for resource rForecourtrS3AccessLogsBucket with message: #/BucketName: expected maxLength: 63, actual: 65

**SDLF release (if known):**
v2.0

**Comment by cnfait** [comment]
Thanks for reporting this! I've investigated the issue and it is not strictly with the organisation name, but more that I've not accounted for longer AWS region names. I'll release a fix soon.

**Comment by cnfait** [comment]
[We've just released SDLF 2.0.1](https://github.com/awslabs/aws-serverless-data-lake-framework/releases/tag/2.0.1) containing a fix for this issue. Thanks again!

If you were following the workshop, I would advise cleaning existing CloudFormation stacks you deployed previously, and start again. If you are following steps under the [Deploying SDLF](https://catalog.us-east-1.prod.workshops.aws/workshops/501cb14c-91b3-455c-a2a9-d0a21ce68114/en-US/10-deployment) section, the workshop is up to date.

If you are following steps under [Hitting production](https://catalog.us-east-1.prod.workshops.aws/workshops/501cb14c-91b3-455c-a2a9-d0a21ce68114/en-US/20-production) instead, make sure to download the new version:
```
curl -L -O https://github.com/awslabs/aws-serverless-data-lake-framework/archive/refs/tags/2.0.1.tar.gz
tar xzf 2.0.1.tar.gz
cd ./aws-serverless-data-lake-framework-2.0.1/
./deploy.sh -h
```

It will be updated soon!
